index.js

```diff
@@ -2,42 +2,66 @@
 var argv = process.argv;
 
 module.exports = (function () {
+	var support = function (level) {
+		if (level <= 0) return null;
+		var result = {};
+		result.level = level;
+		result.has16 = result.hasBasic = level >= 1;
+		result.has256 = level >= 2;
+		result.has16m = level >= 3;
+		return result;
+	}
+
 	if ('FORCE_COLOR' in process.env) {
-		return true;
+		return support(1);
```

Comment:
This doesn't make much sense to me. If `FORCE_COLOR` is true(thy) the level should obviously be `> 0`, but forcing it to `=== 1` seems arbitrary.

I'd wait till all other checks are done, testing if it's still at level `0` and only then forcing it to `1` if `FORCE_COLOR`